locales/en-US/server.ftl

```diff
@@ -280,6 +280,7 @@ settingsDescription = You can sign in with Firefox Accounts to sync all your scr
 settingsPageSubHeader = Sync & Accounts
 settingsClosePreferences =
     .title = Close preferences
+settingsFxaDisconnectDescription = If you disconnect with Firefox Accounts your synced screenshots will remain in your account but you will need to signin to regain access to them.
```

Comment:
If you sign out, you will need to sign in again to regain access to your screenshots.